apps/vast/CHANGELOG.md

```diff
@@ -10,7 +10,11 @@ Every entry has a category for which we use the following visual abbreviations:
 - ⚡️ Breaking Changes
 - 🐞 Bug Fixes
 
-<!-- ## Unreleased -->
+## Unreleased
+
+- 🎁 `pyvast-threatbus` now comes with its own Dockerfile. Pre-built images are
+  available on Dockerhub.
```

Comment:
Please add a link to Dockerhub, e.g., http://hub.docker.com/r/tenzir/pyvast-threatbus.